services/iplant-groups/src/iplant_groups/clients/grouper.clj

```diff
@@ -714,16 +714,15 @@
 (defn- remove-existing-permissions
   [username attribute-def-name & [params]]
   (->> (get-permission-assign-ids username attribute-def-name params)
-       (map (fn [[permission-type ids]] (remove-permission-assign-ids username permission-type ids)))
-       dorun))
+       (mapv (fn [[permission-type ids]] (remove-permission-assign-ids username permission-type ids)))))
 
-(defn- remove-existing-role-permissions
```

Comment:
Yeah, I was ambivalent about that too. I think I'll leave it for now and make the function private in the next PR.